app/dashboard/router.py

```diff
@@ -267,6 +269,17 @@ def get_queryset(self):
             statuses = self.request.query_params.get('status__in').split(',')
             queryset = queryset.filter(idx_status__in=statuses)
 
+        if 'applicants' in param_keys:
+            applicants = self.request.query_params.get('applicants')
```

Comment:
Can we do this more dynamic? since if we decide to add diff values in the future this will be parsed to the query?  something like https://github.com/gitcoinco/web/blob/master/app/dashboard/views.py#L762

Reply:
done